src/main/java/com/example/parser/utils/FileDownload.java

```diff
@@ -0,0 +1,45 @@
+package com.example.parser.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class FileDownload {
+
+    private File tmp;
+    private String fileUrl;
+    private byte[] buffer;
+    private int read = 0;
+    private static final Logger log = LoggerFactory.getLogger(FileDownload.class);
+
+    public FileDownload(String fileUrl) throws IOException {
+        buffer = new byte[1024];
+        tmp = File.createTempFile("dozvoli-na-vikidi-onovlenii", ".tmp", (new File("/Users/kristina/Downloads/")));
```

Comment:
Never do it. This code must work on any environment without code changes.

Reply:
done